API/src/config.ts

```diff
@@ -0,0 +1,11 @@
+export const dbConnection = {
+    database: '',
```

Comment:
this is a good placeholder, but these values should come from ENV vars, not a config file - so maybe you can already build that logic in

Reply:
I think I will add a function to read ENV var and create that object later.

Reply:
@jeff-knurek #399 I made the change for this in the PR